back/key.js

```diff
@@ -0,0 +1,3 @@
+const key = 'lestokenscestbienmaisjwtpasoufpourlasecuritevivelessessions';
```

Comment:
if you're concerned with security, start with not pushing your key in a public repository...